docs/docs-content/integrations/aws-autoscaler.md

```diff
@@ -26,393 +29,781 @@ conditions:
+
+### Usage
+
+The Helm-based Cluster Autoscaler pack is available for Amazon EKS host clusters. To deploy the pack, you must first
+define an IAM policy in the AWS account associated with Palette. This policy allows the Cluster Autoscaler to scale the
+cluster's node groups.
+
+Use the following steps to create the IAM policy and deploy the Cluster Autoscaler pack.
+
+1. In AWS, create a new IAM policy using the snippet below and give it a name, for example,
+   _PaletteEKSClusterAutoscaler_. Refer to the
+   [Creating IAM policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) guide for
+   instructions.
+
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Action": [
+           "autoscaling:DescribeAutoScalingGroups",
+           "autoscaling:DescribeAutoScalingInstances",
+           "autoscaling:DescribeLaunchConfigurations",
+           "autoscaling:DescribeScalingActivities",
+           "autoscaling:DescribeTags",
+           "ec2:DescribeInstanceTypes",
+           "ec2:DescribeLaunchTemplateVersions"
+         ],
+         "Resource": ["*"]
+       },
+       {
+         "Effect": "Allow",
+         "Action": [
+           "autoscaling:SetDesiredCapacity",
+           "autoscaling:TerminateInstanceInAutoScalingGroup",
+           "ec2:DescribeImages",
+           "ec2:GetInstanceTypesFromInstanceRequirements",
+           "eks:DescribeNodegroup"
+         ],
+         "Resource": ["*"]
+       }
+     ]
+   }
+   ```
+
+2. Copy the IAM policy
+   [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html). Your policy ARN
+   should be similar to `arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler`.
+
+3. During the cluster profile creation, modify the `managedMachinePool.roleAdditionalPolicies` section in the
+   **values.yaml** file of the Kubernetes pack with the created IAM policy ARN. Palette will attach the IAM policy to
+   your cluster's node group during cluster deployment. The snapshot below illustrates the specific section to update
+   with the policy ARN.
+
+   ![A snapshot displaying the ARN added to the Kubernetes pack's manifest.](/integrations_aws-cluster-autoscaler_k8s-manifest.webp)
+
+   For example, the code block below displays the updated `managedMachinePool.roleAdditionalPolicies` section with a
+   sample policy ARN.
+
+   ```yaml
+   managedMachinePool:
+     # roleName: {{ name of the self-managed role | format "${string}" }}
+     # A list of additional policies to attach to the node group role
+     roleAdditionalPolicies:
+       - "arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler"
```

Comment:
Same as above, let's replace the account ID

Reply:
:white_check_mark: